ouroboros-network/test/Test/Ouroboros/Network/PeerSelection/Instances.hs

```diff
@@ -81,6 +86,40 @@ instance Arbitrary PeerSelectionTargets where
     , let targets' = PeerSelectionTargets r' k' e' a'
     , sanePeerSelectionTargets targets' ]
 
+instance Arbitrary DomainAddress where
+  arbitrary =
+    DomainAddress . encodeUtf8
+      <$> elements domains
+      <*> (fromIntegral <$> (arbitrary :: Gen Int))
+    where
+      domains = [ "test1"
+                , "test2"
+                , "test3"
+                , "test4"
+                , "test5"
+                ]
```

Comment:
Are we using just five to make it more likely to draw the same name twice, i.e. when one is generating a list of `DomainAddress`-es?

Reply:
I think that is a good reason, however I have to admit that I just followed what was done over at cardano-node (at `cardano-node/test/Test/Cardano/Node/Gen.hs`), but with less fun naming! 

 https://hackage.haskell.org/package/hedgehog-corpus-0.2.0/docs/src/Hedgehog.Corpus.html#cooking 😛

Reply:
Ha ha, that's a good one, I didn't know such a thing existed.